### ISSUE TITLE
problem running unit tests on Linux

### ISSUE THREAD

**Issue by rmandelb** [GalSim-developers/GalSim#895]
On a Linux cluster that I regularly use for running GalSim, I just noticed that on the master branch, it's failing to run the unit tests.  The output at the end looks like
```
Starting python tests...
Nosetests terminated by signal  11
scons: *** [tests/tests.log] Explicit exit, status -11
scons: building terminated because of errors.
```

It is fine on the releases/1.4 branch.  Any ideas offhand, or should I use ```git bisect``` to try to track down where the change occurred?

**Comment by rmjarvis** [comment]
Usually that's some kind of binary incompatibility related to linking to the wrong library.  (signal 11 is a seg fault.)  You can try bisecting, but I'd probably start by checking which version is imported when you do `import galsim` and check that the library in that directory is the library that you just compiled, etc.

**Comment by rmandelb** [comment]
You are right, it was linking to the wrong library.

**Comment by rmandelb** [comment]
Closed #895.
